data/campaigns/tutorial/scenarios/01_Tutorial_part_1.cfg

```diff
@@ -173,6 +173,22 @@
         ) (
             {PRINT ( _ "Click on Li’sar")}
         )}
+
+        # For undo warning always keep a list of target hexes in target_loc
+        # First find Delfador...
+        [store_unit]
+            [filter]
+                side=3
+            [/filter]
+            variable=target_unit
+        [/store_unit]
+        # ...then let "check_undo" event use the hexes around him, stored in target_loc
+        [store_locations]
+            variable=target_loc
+            x=$target_unit.x
+            y=$target_unit.y
+            radius=1          # hexes next to Delfador
+        [/store_locations]
```

Comment:
You can use [filter] inside [sore_locations] to filter for units.

Reply:
Did not know that, will test it in the evening, thanks.